Improve GL JS examples that use the Geocoding API (#525)
mapbox/mapbox-gl-js-docs: docs/pages/example/mapbox-gl-geocoder-custom-render.md

- "import iframe from './mapbox-gl-geocoder-custom-render.html?iframe';"
---

> This example uses a custom HTML rendering function to customize the dropdown menu for a geocoding control.
> 
> It uses the [mapbox-gl-geocoder](https://github.com/mapbox/mapbox-gl-geocoder) control, which enables places search using the [Mapbox Geocoding API](https://docs.mapbox.com/api/search/geocoding/).

Reviewer: ```suggestion
This example uses a custom HTML rendering function to customize the [mapbox-gl-geocoder](https://github.com/mapbox/mapbox-gl-geocoder) dropdown menu.

The [mapbox-gl-geocoder](https://github.com/mapbox/mapbox-gl-geocoder) plugin enables place search using the [Mapbox Geocoding API](https://docs.mapbox.com/api/search/geocoding/).
```
